src/SecureHeaders.php

```diff
@@ -1246,30 +1189,32 @@ private function deconstructHeaderValue(
         return $attributes;
     }
 
-    private function validateHeaders()
+    private function validateHeaders(HeaderBag $headers)
```

Comment:
I'll patch it up (can reuse some of the code in CompileCSP by making the policy decomposition static 😄 )